### DISCUSSION TITLE
Can we implement lyra with mongodb.

### DISCUSSION THREAD

**Discussion by 0x2f0** [oramasearch/orama#207] [Q&A] [answered]
lets say for example you have your data store in mongodb database, Now you want to make a full text search to that data in the database is it possible using lyra? if so how ?

**Comment by cope** [comment, marked as answer]
I'm not an expert, but the way I understood it is that you would have to copy your db data into orama/lyra and then search on that data...

In other words, yes - duplicated. But, as far as I'm aware, that's how elasticsearch works too. (also not an expert there either)

**Reply by micheleriva** [reply]
Yes, @cope is correct
